.github/scripts/test-all.sh

```diff
@@ -17,7 +17,7 @@ echo "$packages" | tr ' ' '\n' | while read -r item; do
   echo "running $item"
 
   case "$item" in
-    *".github"*|*"experimental"*|*"deployment-platforms"*|"typescript/subscriptions-pubsub")
+    *".github"*|*"experimental"*|*"deployment-platforms"*|*"typescript/subscriptions-pubsub"*|*"typescript/postgis-express"*|*"typescript/graphql"*|*"typescript/graphql-nextjs"*|*"javascript/graphql"*|*"javascript/graphql-apollo-server"*|*"javascript/graphql-auth"*)
```

Comment:
What is the reasoning behind ignoring all these?

Reply:
`*".github"*|*"experimental"*|*"deployment-platforms"*` I don't know, that is how it was when I picked it. 

Rest use `nexus-prisma` which is broken at the moment.

Reply:
Actually, we can keep those after all since we "fixed" the corresponding tests by reverting the versions of the `@prisma` packages back to `2.7.0`. Let's rather keep them in the tests and resolve the issues by reverting to a lower version rather than removing tests. (Sorry I think I wasn't clear about this before).

Reply:
Noted, will do so in a new PR.

Reply:
Removed all the nexus ones via https://github.com/prisma/prisma-examples/pull/2079

The "postgis" one is most likely something else (I need to investigate). Will remove that in a separate PR.